pkg/config/loader.go

```diff
@@ -48,7 +48,7 @@ func LoadFromViper(viperCfg func(v *viper.Viper)) DataStoreLoaderFunc {
 	return func(ctx context.Context, cfg *Config, templateData map[string]interface{}) error {
 		home, _ := cfg.GetHomeDir()
 
-		ctx, log := tracing.StartSpanWithName(ctx, "LoadFromViper", attribute.String("porter.PORTER_HOME", home))
+		_, log := tracing.StartSpanWithName(ctx, "LoadFromViper", attribute.String("porter.PORTER_HOME", home))
```

Comment:
I'd prefer to not change this here because setting the ctx variable is important. Even if ctx isn't used after this point, if we add a line in the future that uses ctx, with this change they will accidentally use the _old_ ctx instead of the new scoped context.

I see more changes like this below and really all of those changes need to be reverted for the same reason.

Reply:
Makes sense. I'll revert it and add the linter warning.